mlfinlab/labeling/return_vs_benchmark.py

```diff
@@ -0,0 +1,39 @@
+"""
+Return in excess of a given benchmark
+
+Chapter 5, Machine Learning for Factor Investing, by Coqueret and Guida, (2020).
+"""
+import warnings
+import numpy as np
+import pandas as pd
+
+
+def return_over_benchmark(returns, benchmark=0, binary=False):
+    """
+    Return over benchmark labeling method. Sourced from Chapter 5.5.1 of Machine Learning for Factor Investing,
+    by Coqueret, G. and Guida, T. (2020).
+
+    Returns a Series or DataFrame of numerical or categorical returns over a given benchmark. The time index of the
+    benchmark must match those of the price observations.
+
+    :param returns: (pd.Series or pd.DataFrame) Time indexed prices to find labels for. NaN values are fine and will
+                simply result in a NaN label. Use np.pct_change() or similar to get returns from prices.
+    :param benchmark: (pd.Series or float) Benchmark prices to compare the given prices against for labeling. Can be a
```

Comment:
Prices should be used as input to be consistent with other labeling techniques: PR #342, PR #351, PR #357